pkg/controller/release/release.go

```diff
@@ -42,6 +44,13 @@ import (
 	helmClient "github.com/crossplane-contrib/provider-helm/pkg/clients/helm"
 )
 
+const (
+	maxConcurrency = 10
```

Comment:
Just curious, what made you go with 10 here?

Reply:
Nothing special, I just some number bigger than 1 (e.g. concurrent), not to block other release's because of a 
(or some) long running one (e.g. with a pre-install hook). 
I believe we may end up having this configurable somehow but for now, just trying to set some reasonable default.